src/pages/home/report/ReportActionCompose.js

```diff
@@ -650,12 +648,11 @@ class ReportActionCompose extends React.Component {
                                         shouldClear={this.state.textInputShouldClear}
                                         onClear={() => this.setTextInputShouldClear(false)}
                                         isDisabled={isComposeDisabled || isBlockedFromConcierge}
-                                        selection={this.state.selection}
                                         onSelectionChange={this.onSelectionChange}
                                         isFullComposerAvailable={this.state.isFullComposerAvailable}
                                         setIsFullComposerAvailable={this.setIsFullComposerAvailable}
                                         isComposerFullSize={this.props.isComposerFullSize}
-                                        value={this.state.value}
+                                        defaultValue={this.props.comment}
```

Comment:
`props.comment` will update on each keystroke. Is it fine to directly assign it to the prop?

Note: (sorry for the confusing comment, I don't know how to explain it properly) There is a strange bug on RN where RN keeps the value after the cursor in view and updates the text before the cursor when the user is typing in the middle of the text. On the other hand, native text input on iOS has the opposite behavior to this. 

I will try to find the issue and link it here and also test it.

Reply:
> `props.comment` will update on each keystroke. Is it fine to directly assign it to the prop?

`props.comment` should update only after sending a message, on `submit`

Reply:
Yes @kidroca you are right, there is no use for it and I cleaned it up.

In regards to the cursor issue you described @parasharrajat that shouldn't be an issue as updates to the `defaultValue` will be ignored / only be considered on the first render